src/Umbraco.Web/Runtime/WebInitialComposer.cs

```diff
@@ -108,6 +108,7 @@ public override void Compose(Composition composition)
             composition.RegisterUnique<HtmlUrlParser>();
             composition.RegisterUnique<HtmlImageSourceParser>();
             composition.RegisterUnique<RichTextEditorPastedImages>();
+            composition.RegisterUnique<PropertyEditors.ValueConverters.BlockEditorConverter>();
```

Comment:
Any reason registered unique here & not put into the `PropertyValueConverterCollection`  on line 177?

Reply:
This is not a property value converter, it is just a service that a couple of them depend on. Make sense?

Reply:
I think the more verbose namespace made me think it was a ValueConvertor itself just because of where it lives @Shazwazza